frontend/ui/src/cookies/cookie.ts

```diff
@@ -1,34 +0,0 @@
-export function setCookie(name: string, value: string, domain: string, days = 3650) {
```

Comment:
The authentication cookie will be an httpOnly cookie, which means that the frontend does not have direct access and will call `/users.getPrincipal` instead to get the users' information